index-handlers/src/test/java/no/sikt/nva/nvi/index/SearchNviCandidatesHandlerTest.java

```diff
@@ -1,6 +1,7 @@
 package no.sikt.nva.nvi.index;
```

Comment:
Also update test `shouldReturnPaginatedSearchResultWithCorrectQueryParamsFilterAndQueryInIdIfGiven()` with title?

Reply:
You can rename it to `shouldReturnPaginatedSearchResultWithCorrectQueryParamsInIdIfGiven()`